### PR TITLE
[AdminListBundle] Enhancement: Streamline tests

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | --

This PR

* [x] is based on https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/2813#issuecomment-722183497
* [x] removes useless comments
* [x] fixes namespaces
* [x] removes `unit` folder to not violate PSR-4
* [x] replaces #2813 
* [x] removes not used code parts with a class `\UnitTester` which is not available 🤔  :
```php
    /**
     * @var \UnitTester
     */
    protected $tester;
```

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* This PR seems to need a milestone of a patch release.

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

File: src/Kunstmaan/AdminListBundle/Tests/Service/EntityVersionLockServiceTest.php [hunk 1]
```diff
@@ -6,15 +6,12 @@
...
+use Kunstmaan\AdminListBundle\Tests\Model\TestLockableEntityInterfaceImplementation;
 use PHPUnit\Framework\TestCase;
 
-/**
- * class EntityVersionLockServiceTest
- */
 class EntityVersionLockServiceTest extends TestCase
 {
-    protected static $TEST_CLASS = 'Kunstmaan\\AdminListBundle\\Tests\\unit\\Model\\TestLockableEntityInterfaceImplementation';
+    protected static $TEST_CLASS = 'Kunstmaan\\AdminListBundle\\Tests\\\Model\\TestLockableEntityInterfaceImplementation';
```
Comment (acrobat) [inline src/Kunstmaan/AdminListBundle/Tests/Service/EntityVersionLockServiceTest.php:14]:
```suggestion
    protected static $TEST_CLASS = 'Kunstmaan\\AdminListBundle\\Tests\\Model\\TestLockableEntityInterfaceImplementation';
```

One `\` too many

**Review by acrobat — CHANGES_REQUESTED** [review]
Can you check the failing styleci check? Travis also fails but will be fixed I think with the suggested code change

**Comment by OskarStark** [comment]
Sure will check tomorrow

File: src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/DateTimeFilterTypeTest.php [hunk 2]
```diff
@@ -9,10 +9,7 @@
 
 class DateTimeFilterTypeTest extends BaseDbalFilterTest
 {
-    /**
-     * @var \UnitTester
-     */
-    protected $tester;
+
```
Comment (OskarStark) [inline src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/DateTimeFilterTypeTest.php:12]:
```suggestion
```

File: src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/EnumerationFilterTypeTest.php [hunk 3]
```diff
@@ -5,15 +5,9 @@
...
-/**
- * Generated by PHPUnit_SkeletonGenerator 1.2.1 on 2014-02-26 at 15:06:04.
- */
 class EnumerationFilterTypeTest extends BaseDbalFilterTest
 {
-    /**
-     * @var \UnitTester
-     */
-    protected $tester;
+
```
Comment (OskarStark) [inline src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/EnumerationFilterTypeTest.php:10]:
```suggestion
```

File: src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/StringFilterTypeTest.php [hunk 4]
```diff
@@ -5,15 +5,9 @@
...
-/**
- * Generated by PHPUnit_SkeletonGenerator on 2012-09-26 at 13:21:33.
- */
 class StringFilterTypeTest extends BaseDbalFilterTest
 {
-    /**
-     * @var \UnitTester
-     */
-    protected $tester;
+
```
Comment (OskarStark) [inline src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/StringFilterTypeTest.php:10]:
```suggestion
```

File: src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/NumberFilterTypeTest.php [hunk 5]
```diff
@@ -5,15 +5,9 @@
...
-/**
- * Generated by PHPUnit_SkeletonGenerator on 2012-09-26 at 13:21:33.
- */
 class NumberFilterTypeTest extends BaseDbalFilterTest
 {
-    /**
-     * @var \UnitTester
-     */
-    protected $tester;
+
```
Comment (OskarStark) [inline src/Kunstmaan/AdminListBundle/Tests/AdminList/FilterType/DBAL/NumberFilterTypeTest.php:10]:
```suggestion
```

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @OskarStark!

**Pull request merged**